r_arm:
  type: JointSplineTrajectoryController
  joints:
    - r_shoulder_pan_joint
    - r_shoulder_lift_joint
    - r_upper_arm_roll_joint
    - r_elbow_flex_joint
    - r_forearm_roll_joint
    - r_wrist_flex_joint
    - r_wrist_roll_joint
  gains:
    r_shoulder_pan_joint: {p: 120.0, d: 20.0}
    r_shoulder_lift_joint: {p: 150.0, d: 10.0}
    r_upper_arm_roll_joint: {p: 150.0, d: 7.0}
    r_elbow_flex_joint: {p: 100.0, d: 5.0}
    r_forearm_roll_joint: {p: 100.0, d: 10.0}
    r_wrist_flex_joint: {p: 100.0, d: 8.0}
    r_wrist_roll_joint: {p: 100.0, d: 8.0}


l_arm:
  type: JointSplineTrajectoryController
  joints:
    - l_shoulder_pan_joint
    - l_shoulder_lift_joint
    - l_upper_arm_roll_joint
    - l_elbow_flex_joint
    - l_forearm_roll_joint
    - l_wrist_flex_joint
    - l_wrist_roll_joint
  gains:
    l_shoulder_pan_joint: {p: 120.0, d: 20.0}
    l_shoulder_lift_joint: {p: 150.0, d: 10.0}
    l_upper_arm_roll_joint: {p: 150.0, d: 7.0}
    l_elbow_flex_joint: {p: 100.0, d: 5.0}
    l_forearm_roll_joint: {p: 100.0, d: 10.0}
    l_wrist_flex_joint: {p: 100.0, d: 8.0}
    l_wrist_roll_joint: {p: 100.0, d: 8.0}
